barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_verifier.cpp

```diff
@@ -72,12 +72,13 @@ std::shared_ptr<typename DeciderVerificationKeys::DeciderVK> ProtogalaxyVerifier
 {
     static constexpr size_t BATCHED_EXTENDED_LENGTH = DeciderVerificationKeys::BATCHED_EXTENDED_LENGTH;
     static constexpr size_t NUM_KEYS = DeciderVerificationKeys::NUM;
-
-    const std::shared_ptr<const DeciderVK>& accumulator = keys_to_fold[0];
-    const size_t log_circuit_size = static_cast<size_t>(accumulator->verification_key->log_circuit_size);
+    static constexpr size_t COMBINER_LENGTH = BATCHED_EXTENDED_LENGTH - NUM_KEYS;
 
     run_oink_verifier_on_each_incomplete_key(proof);
 
```

Comment:
A shared ptr to a copy of the first key to be folded is created. Doing it this way helps to avoid using next_accumulator. Tried mutating in place, but it breaks recursive verification